indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -940,17 +943,9 @@ public SeekableStreamSupervisor(
         spec.isSuspended()
     );
 
-    int workerThreads;
+    final int workerThreads = calculateWorkerThreads(tuningConfig, ioConfig);
     if (autoScalerConfig != null && autoScalerConfig.getEnableTaskAutoScaler()) {
       log.info("Running Task autoscaler for supervisor[%s] for datasource[%s]", supervisorId, dataSource);
```

Comment:
Nit: I guess the log line can move to the new calculate method itself.

Reply:
@kfaraz logging that in a method meant to be calculating worker threads seems a bit odd no?

Reply:
Yeah, I guess you are right. You can leave it as it is then.